includes/abstracts/abstract-wp-job-manager-form.php

```diff
@@ -533,6 +533,35 @@ protected function get_posted_fields() {
 		return apply_filters( 'job_manager_get_posted_fields', $values, $this->fields );
 	}
 
+	/**
+	 * Checks whether a value is empty.
+	 *
+	 * @param string|numeric|array|boolean $value The value that is being checked.
+	 * @param string                       $key   The key of the field that is being checked.
+	 * @return bool True if value is empty, false otherwise.
+	 */
+	protected function is_empty( $value, $key = '' ) {
```

Comment:
Doubt anyone is using this, but setting default to `$key` because it isn't part of initial implementation. We could add a deprecation notice for ``, but not sure if necessary.